.dockerignore

```diff
@@ -0,0 +1,69 @@
+# same as .gitignore
```

Comment:
I had this on my laptop since a while apparently, it's just a copy of .gitignore. could be overhauled I guess, but meeh